src/testing/weather.test.js

```diff
@@ -0,0 +1,25 @@
+describe('Selectors in weatherSlice', () => {
+  const testData = {
+    weather: [
+      { main: { temp: 27.44 } },
+      { name: 'Pakistan' },
+    ],
+    Status: false,
+    error: null,
+  };
+
+  it('Weather should return the weather data', () => {
+    const selectedData = testData.weather;
+    expect(selectedData).toEqual(testData.weather);
+  });
+
+  it('MockState should return the weather status', () => {
+    const selectedStatus = testData.Status;
+    expect(selectedStatus).toBe(testData.Status);
+  });
+
+  it('MockState should return the weather error', () => {
```

Comment:
- [Optional] Dear @sheezu900, Please notice this test is not really testing the application, It's not much of a value having these tests here. 
- [ ] Also Just as mentioned earlier, you have not added integration tests for the project. As stated in the project requirements, we need to make use of `integration tests using React Testing Library`.  This will help your test to behave as a real user and test your application on how it interacts. 

> This is sample of a react testing library code 
```JavaScript
import {render, screen} from '@testing-library/react'

render(<MyComponent />)
const lastNameInput = screen.getByDisplayValue('Norris')
```

> Please refer to the React testing library [documentation](https://testing-library.com/docs/react-testing-library/intro/) to learn how to do this.

**The whole point is to mimic a user interaction and assert the app works as expected.**

Reply:
can u help me i dont know whats to do please